lib/runtime/application.ts

```diff
@@ -195,12 +198,24 @@ export default class Application extends Addon {
    * config/environment.js
    */
   private async createServer(port: number): Promise<void> {
-    await new Promise((resolve) => {
+      await new Promise((resolve) => {
```

Comment:
Derp - apparently still have some gaps in my vscode config lol ...

It shouldn't _actually_ conflict since the more local var will shadow the parent scope, but I'll rename for clarity.